hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -195,41 +197,50 @@ public Pair<ResponseCodeEnum, Long> create(
                     "Cannot auto-create an account from unaliased change " + change);
         }
 
-        // checks tokenAliasMap if the change consists an alias that is already used in previous
-        // iteration of the token transfer list. This map is used to count number of
-        // maxAutoAssociations needed on auto created account
-        analyzeTokenTransferCreations(changes);
+        TransactionBody.Builder syntheticCreation;
+        String memo;
+        HederaAccountCustomizer customizer = new HederaAccountCustomizer();
+        if (alias.size() == EntityIdUtils.EVM_ADDRESS_SIZE) {
+            syntheticCreation = syntheticTxnFactory.createHollowAccount(alias, 0L);
+            customizer.key(EMPTY_KEY);
+            memo = LAZY_MEMO;
```

Comment:
Shouldn't `maxAutoAssociations` be set on the created account for associating the transferred tokens, if we are creating account with token transfers to `evmAddress` ?

Reply:
Yes, it should, but as [discussed above](https://github.com/hashgraph/hedera-services/pull/4004#issuecomment-1283954379) we will handle the token transfers in [a separate PR](https://github.com/hashgraph/hedera-services/pull/4136).